bigframes/dataframe.py

```diff
@@ -3472,6 +3472,52 @@ def _apply_unary_op(self, operation: ops.UnaryOp) -> DataFrame:
         block = self._block.multi_apply_unary_op(self._block.value_columns, operation)
         return DataFrame(block)
 
+    def f(
```

Comment:
IIUC, it's internal function. Maybe `_f` or other more readable name.

Reply:
I didn't see this method gets called. Please double check.

Reply:
accidentally copied `_map_clustering_columns` removed in new revision